criteria/open-standards.md

```diff
@@ -9,6 +9,7 @@ order: 10
 * For features of a codebase that facilitate the exchange of data the codebase MUST use an open standard that meets the [Open Source Initiative Open Standard Requirements](https://opensource.org/osr)
 * If no existing open standard is available, effort SHOULD be put into developing one
 * Standards that are machine testable SHOULD be preferred over those that are not
+* Codebases SHOULD include a [publiccode.yml](https://github.com/publiccodenet/publiccode.yml/blob/master/README.md) metadata description so that they're easily discoverable
```

Comment:
```suggestion
* Codebases SHOULD include a [publiccode.yml](https://github.com/publiccodenet/publiccode.yml/) metadata description so that they're easily discoverable
```

You don't need to specifically mention the file in the `master` branch.